dashboard/app/views/levels/_single_multi.html.haml

```diff
@@ -1,10 +1,10 @@
-.multi{id: "multi_#{index}"}
+- data = level.properties
+- app = 'multi'
 
-  - data = level.properties
-  - app = 'multi'
+- # setting this so that multi_t works
+- @level = level
 
-  - # setting this so that multi_t works
-  - @level = level
+.multi{id: "level_#{index}"}
```

Comment:
I was going to suggest defining a method to do this since it happens several times here and in single_text_match but then I remember that there's a rails helper: http://api.rubyonrails.org/classes/ActionView/RecordIdentifier.html#method-i-dom_id

I checked that it does use the base class, Level, rather than the STI class (like Multi):
````
irb(main):003:0> dom_id(Level.last)
  Level Load (2.2ms)  SELECT  `levels`.* FROM `levels`  ORDER BY `levels`.`id` DESC LIMIT 1
=> "level_5361"
irb(main):004:0> dom_id(Level.first)
  Level Load (1.0ms)  SELECT  `levels`.* FROM `levels`  ORDER BY `levels`.`id` ASC LIMIT 1
=> "level_1"
irb(main):005:0> dom_id(Multi.last)
  Multi Load (1.3ms)  SELECT  `levels`.* FROM `levels` WHERE `levels`.`type` IN ('Multi')  ORDER BY `levels`.`id` DESC LIMIT 1
=> "level_5234"
````